geode-core/src/main/java/org/apache/geode/distributed/internal/streaming/StreamingOperation.java

```diff
@@ -501,15 +512,24 @@ public void fromData(DataInput in) throws IOException, ClassNotFoundException {
       this.pdxReadSerialized = in.readBoolean();
       Version senderVersion = InternalDataSerializer.getVersionForDataStream(in);
       boolean isSenderAbove_8_1 = senderVersion.compareTo(Version.GFE_81) > 0;
+      InternalCache cache = null;
+      Boolean initialPdxReadSerialized = false;
+      try {
+        cache =
+            (InternalCache) GemFireCacheImpl.getForPdx("fromData invocation in StreamingOperation");
+        initialPdxReadSerialized = cache.getPdxReadSerializedOverride();
+      } catch (CacheClosedException e) {
+
```

Comment:
Maybe we should add a log here. since we are consuming the exception silently.
Also i was wondering if we could just return from here. If the cache itself is closed there maybe no point in executing the below code. maybe.

Reply:
done